fbfmaproom/fbfmaproom.py

```diff
@@ -735,16 +783,18 @@ def display_prob_thresh(val):
     Input("location", "pathname"),
     Input("severity", "value"),
     Input("obs_datasets", "value"),
+    State("obs_state", "data"),
     State("season", "value"),
 )
-def _(issue_month_idx, freq, mode, geom_key, pathname, severity, obs_dataset_key, season):
+def _(issue_month_idx, freq, mode, geom_key, pathname, severity, obs_dataset_key, obs_state, season):
     country_key = country(pathname)
     config = CONFIG["countries"][country_key]
     tcs = table_columns(config["datasets"]["observations"], obs_dataset_key)
```

Comment:
This was a leftover because the two output entirely different formats. Removed the old function since it turned out to be unnecessary.

Side: in the download_table endpoint, table_columns is called and the result is never used. Should this be deleted?